src/Symfony/Component/Yaml/Parser.php

```diff
@@ -634,6 +643,11 @@ private function isCurrentLineComment()
         return '' !== $ltrimmedLine && $ltrimmedLine[0] === '#';
     }
 
+    private function isCurrentLineLastLineInDocument()
```

Comment:
Let's "nitpick" a bit: would `isLastDocumentLine()` be a better name for `isCurrentLineLastLineInDocument()`?